safety/src/main/java/com/hms/lib/commonmobileservices/safety/google/GoogleSafetyMapper.kt

```diff
@@ -19,6 +19,6 @@ import com.hms.lib.commonmobileservices.safety.common.Mapper
 
 class GoogleSafetyMapper: Mapper<SafetyNetApi.RecaptchaTokenResponse, SafetyServiceResponse>() {
     override fun map(from: SafetyNetApi.RecaptchaTokenResponse): SafetyServiceResponse = SafetyServiceResponse(
-        responseToken = from.tokenResult
+        responseToken = from.tokenResult!!
```

Comment:
Could you please check that the return type is non-nullable but the method returns an unchecked value?

Reply:
resolved